### ISSUE TITLE
In plot_techmix() some arguments could be named more simply

### ISSUE THREAD

**Issue by maurolepore** [2DegreesInvesting/r2dii.plot.static#115]
The prefix df_ cominicates something that is alrady documented in the
parameters description, i.e. that the argument takes a data frame. 

Also, the 'tech' bit in df_tech_colours gives information that is
already available from the context of the argument, i.e. a function
that contains 'tech' in the name. 

Finally, df_bar_specs could be named just specs as the users can
learn that all plot_ functions have a specific specs object that
could be used to tweak the plot. 

In conclusion, I propose these new names

df_tech_colours -> colours
df_bar_spects -> specs
